test/contrib/test_contrib.py

```diff
@@ -61,12 +61,19 @@ def test_smoke(self, device, dtype, B, H, D, image_size):
         for f in feats:
             assert f.shape == (B, T, D)
 
+    @pytest.mark.parametrize("H", [3, 8])
+    @pytest.mark.parametrize("D", [245, 1001])
+    @pytest.mark.parametrize("image_size", [32, 224])
+    def test_exception(self, device, dtype, H, D, image_size):
+        with pytest.raises(ValueError):
+            kornia.contrib.VisionTransformer(image_size=image_size, num_heads=H, embed_dim=D).to(device, dtype)
+
     def test_backbone(self, device, dtype):
         def backbone_mock(x):
             return torch.ones(1, 128, 14, 14, device=device, dtype=dtype)
 
         img = torch.rand(1, 3, 32, 32, device=device, dtype=dtype)
-        vit = kornia.contrib.VisionTransformer(backbone=backbone_mock).to(device, dtype)
+        vit = kornia.contrib.VisionTransformer(backbone=backbone_mock, num_heads=8).to(device, dtype)
```

Comment:
doesn't it make sense to parametrize to test with different `num_heads`?

Reply:
I can add it, but I'm not sure whether it is needed - test_backbone corresponds to the test of custom backbone network for ViT, and different number of heads is tested in the test_smoke function. If you say so, I can parametrize num_heads here also.

Reply:
oh, I didn't see it's already on others' tests. It's fine this way :)